pom.xml

```diff
@@ -74,6 +74,11 @@
 			<artifactId>logback-classic</artifactId>
 			<scope>runtime</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.experimental</groupId>
+			<artifactId>spring-graal-native-feature</artifactId>
+			<version>0.6.0.BUILD-SNAPSHOT</version>
+		</dependency>
```

Comment:
Can the scope of this be different from `compile`?

Reply:
Probably runtime would work, since it isn't actually needed to compile. I think @making would have some other ideas how to set up the Maven build.